pyiron/atomistics/structure/analyse.py

```diff
@@ -0,0 +1,133 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+import numpy as np
+from pyiron_base import Settings
+from sklearn.cluster import AgglomerativeClustering
+from scipy.sparse import coo_matrix
+
+__author__ = "Joerg Neugebauer, Sam Waseda"
+__copyright__ = (
+    "Copyright 2020, Max-Planck-Institut für Eisenforschung GmbH - "
+    "Computational Materials Design (CM) Department"
+)
+__version__ = "1.0"
+__maintainer__ = "Sam Waseda"
+__email__ = "waseda@mpie.de"
+__status__ = "production"
+__date__ = "Sep 1, 2017"
+
+s = Settings()
+
+def get_average_of_unique_labels(labels, values):
```

Comment:
Is this part of this PR or #1220?

Reply:
Both. I think it's more appropriate to have it in `analyse.py` and I would import it in `neighbors.py` (so I'll delete the one in `neighbors.py` as soon as this one gets merged)